ballot/admin/ballot.py

```diff
@@ -2,4 +2,9 @@
 
 from ..models import ballot as models
 
-admin.site.register(models.Ballot)
+
+class BallotAdmin(admin.ModelAdmin):
+    exclude = ['locality']
```

Comment:
It's not; it's a generic `Locality`, which can refer to any of those.

We need to filter the `Locality` values that are shown in the dropdown, not to eliminate the dropdown. I would suggest filtering out ZipCode instances.

Perhaps something like this:
https://stackoverflow.com/questions/6581520/filtering-dropdown-values-in-django-admin#answer-6582108